<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Silent Edge</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="MainMenu">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>100</number>
        </property>
        <property name="rightMargin">
         <number>100</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonCreateServer">
          <property name="text">
           <string>Create Server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonJoinGame">
          <property name="text">
           <string>Join Game</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSettings">
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonQuit">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_settings">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>499</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="ip_label">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>IP adress:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ip_line"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="music_label">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Music:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_music">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButtonBack">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButtonDone">
            <property name="text">
             <string>Done</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_server_list">
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
        <property name="rightMargin">
         <number>44</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="leftMargin">
           <number>14</number>
          </property>
          <property name="rightMargin">
           <number>235</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Enter server ip:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="serverIP_line"/>
          </item>
          <item row="2" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_6"/>
          </item>
          <item row="5" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_5"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_username">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLineEdit" name="lineUsername"/>
          </item>
          <item row="12" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="14" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2"/>
          </item>
          <item row="15" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout"/>
          </item>
          <item row="16" column="0">
           <widget class="QPushButton" name="pushButtonConnect">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item row="16" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3"/>
          </item>
          <item row="17" column="0">
           <widget class="QPushButton" name="pushButtonBack_2">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
